lib/network.sh

```diff
@@ -69,6 +68,19 @@ network_address dhcp_range_end "$PROVISIONING_NETWORK" 100
 
 export CLUSTER_DHCP_RANGE=${CLUSTER_DHCP_RANGE:-"$dhcp_range_start,$dhcp_range_end"}
 
-export EXTERNAL_SUBNET=${EXTERNAL_SUBNET:-"192.168.111.0/24"}
+export IP_STACK=${IP_STACK:-"v4"}
+if [[ "${IP_STACK}" == "v4" ]]; then
+    export EXTERNAL_SUBNET_V4=${EXTERNAL_SUBNET_V4:-"192.168.111.0/24"}
```

Comment:
Would it be worth still supporting EXTERNAL_SUBNET for backwards compatibility, in the default v4 case?

Something like:

```
    export EXTERNAL_SUBNET_V4=${EXTERNAL_SUBNET_V4:-${EXTERNAL_SUBNET:-"192.168.111.0/24"}}
```

Or alternatively perhaps we should fail with a helpful message if folks still have EXTERNAL_SUBNET set in their `env`?

Reply:
I'll push a change to add a helpful message.  Previously EXTERNAL_SUBNET could be v4 or v6, so it can't be used as the default without a little bit more validation than what you suggested.  I don't think requiring a config change is too much if there's a helpful message.

Thanks for the suggestions!